go/vt/vttablet/tabletmanager/rpc_query.go

```diff
@@ -275,6 +275,18 @@ func (tm *TabletManager) ExecuteFetchAsApp(ctx context.Context, req *tabletmanag
 	return sqltypes.ResultToProto3(result), err
 }
 
+// MysqlSystemMetrics gets system metrics from the MySQL deamon
```

Comment:
For our own sake, I think it's better to frame this as `from the mysqlctl[d]` process since that's what we're really interfacing with here as the `MySQLDaemon` interface implementations. Otherwise readers might think that we're interacting with the `mysqld` itself, which we don't do directly and which we definitely don't do in the case of host/OS metrics.

Reply:
Fixed comment.